src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -157,21 +159,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // Load the fallback catalogues
         $fallbackCatalogues = array();
         $translator = $this->getContainer()->get('translator');
-        if ($translator instanceof Translator) {
-            foreach ($translator->getFallbackLocales() as $fallbackLocale) {
-                if ($fallbackLocale === $locale) {
-                    continue;
-                }
 
-                $fallbackCatalogue = new MessageCatalogue($fallbackLocale);
-                foreach ($transPaths as $path) {
-                    $path = $path.'translations';
-                    if (is_dir($path)) {
-                        $loader->loadMessages($path, $fallbackCatalogue);
-                    }
+        $fallbackLocales = [];
+        if ($translator instanceof DataCollectorTranslator || $translator instanceof LoggingTranslator) {
+            try {
+                $fallbackLocales = $translator->getFallbackLocales();
+            } catch (\Throwable $e) {
+            }
+        } elseif ($translator instanceof Translator) {
+            $fallbackLocales = $translator->getFallbackLocales();
```

Comment:
What about a new interface for this? Or at least actually define this method? Imo. we should not rely on the magic `__call` behavior of these translators.